src/com/lftechnology/java/training/krishna/basics/even_fibonacci/EvenFibonacci.java

```diff
@@ -0,0 +1,58 @@
+package com.lftechnology.java.training.krishna.basics.even_fibonacci;
```

Comment:
And, Do you need to create a package structure for *EvenFibonacci*. It could have been accommodated inside the `basics`.